lang/ar/dialogs.php

```diff
@@ -84,16 +84,32 @@
         'mode_album_help' => 'تضمين الصور من هذا الألبوم المحدد.',
         'mode_stream_help' => 'تضمين جميع صورك العامة كتدفق زمني (الأحدث أولاً).',
         'layout' => 'Layout',
+        'layout_square' => 'Square',
```

Comment:
don't we have those layout in 'gallery.layout' ? :)
The only one missing is filmstrip which could also be added there.

Reply:
Thank you :) 

Resolved in https://github.com/LycheeOrg/Lychee/pull/3834/commits/aad4b70b76a97b5003e768a98ad62bc108ac11f0